test/odbc/constants.h

```diff
@@ -1,17 +1,21 @@
 #ifndef ODBC__constants_H_
 #define ODBC__constants_H_
 
+#include <map>
 #include <string>
 
+using std::map;
 using std::string;
 
 namespace constants{
 
 enum class ServerType {
-  Babel,
-  SQL,
+  MSSQL,
+  PSQL,
```

Comment:
Similarly as in previous comment for constants.cpp

Hard coding these values will require code change to add additional driver.
It is probably not necessary even for current functionality.